### DISCUSSION TITLE
Is it possible to ask renovate to help upgrade to a specific version?

### DISCUSSION THREAD

**Discussion by songkeys** [renovatebot/renovate#8824] [Request Help] [answered]
I don't see this in the docs or issues. 

My requirement is that I can specify a package-version map to tell which gets updated to which version (not the latest).

Is this possible? Thanks in advance!

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#allowedversions

**Reply by songkeys** [reply]
Sorry that I missed this. Thank you so much
